Chat room

genmon opened the pull request.

genmon committed:
First version. Functional, unstyled

genmon committed:
Styles

Remaining bug: scroll behaviour

genmon committed:
Scrolling workaround in Safari

genmon committed:
Add OpenAI capability

genmon committed:
Working on the readme

threepointone commented on templates/chat-room/src/client.tsx:
```diff
@@ -0,0 +1,20 @@
+import "./styles.css";
```
If we're moving all the server stuff to "party", then let's rename "src" to "app"?

genmon committed:
Continuing the readme

genmon committed:
Completed draft

threepointone committed:
pass (most) CI errors

genmon committed:
Updated readme

genmon committed:
Tweak

genmon committed:
Fixing image links

genmon committed:
Formating fixes

threepointone committed:
update deps

threepointone committed:
some cleanups

- the input form isn't dependent on the user
- clean up some types
- user hoisted to module scope

threepointone committed:
Add shouldReply

genmon committed:
Updates shouldReply to use a Cloudflare model

genmon committed:
Adds shouldReply to the README

threepointone committed:
stray log, fix a lint error
